common/remote/sockfile.cpp

```diff
@@ -1835,18 +1835,15 @@ class CRemoteFilteredFileIOBase : public CRemoteBase, implements IRemoteFileIO
                 request.append("\"");
             }
         }
-        bufPos = 0;
     }
     virtual size32_t read(offset_t pos, size32_t len, void * data) override
     {
-        assertex(pos == bufPos);  // Must read sequentially
```

Comment:
Is this no longer required ?
If not then sure, remove all bufPos.  If yes then make bufPos offset_t ?

Reply:
no used other than this [incorrect] assert.

Reply:
but I guess if correct, it does protect again misuse. i.e. ensures serial as it needs to be.
So yes I will put back and make a offset_t.